app/repository/family.py

```diff
@@ -338,6 +335,15 @@ def create(db: Session, family: FamilyCreateDTO, geo_id: int, org_id: int) -> st
             value=family.metadata,
         )
     )
+
+    # Add any collections.
+    for col in set(family.collections):
+        db.flush()
```

Comment:
Let's move the flush outside the loop - as it can be expensive - do we actually need it?

Reply:
Great point. Unnecessary for collections, so have moved it outside the loop.